examples/document-types/README.md

```diff
@@ -0,0 +1,51 @@
+# MarkdownDB Schemas Configuration Tutorial
+
+MarkdownDB empowers you to establish and enforce the structure of your Markdown content using the Zod library for schema definitions. This tutorial guides you through the process of defining schemas for a blog post, ensuring a mandatory `date` field. The schemas play a crucial role in error prevention and ensure the predicted structure in Markdown files.
+
+## Define Your Schemas
+
+Create a configuration object and use the Zod library to define your schemas. For more information on defining Zod schemas, visit [Zod Documentation - Basic Usage](https://zod.dev/?id=basic-usage). In this example, we'll begin by creating a schema for a blog post.
+
+Subsequently, we'll use the `client.indexFolder` method along with the defined schemas to scan and index the folder containing your Markdown files.
+
+```javascript
+// index.js
+import { z } from "zod";
+import { MarkdownDB } from "mddb";
+
+// Code for initializing the connection
+const client = /_ Initialize your connection here _/;
```

Comment:
i would do this small piece of work for them - otherwise i'm not clear what is happening.

I'd also do this after defining the schema.